src/accapi/structs.py

```diff
@@ -29,8 +51,8 @@ def __init__(self, *args):
         self.sessionIndex = args.pop(0)
         self.sessionType = SESSION_TYPE[args.pop(0)]
         self.sessionPhase = SESSION_PHASE[args.pop(0)]
-        self.sessionTime = args.pop(0) * 1000
-        self.sessionEndTime = args.pop(0) * 1000
+        self.sessionTime = args.pop(0)
```

Comment:
This was a silly math mistake. The goal was to have all times be represented in seconds, therefor it should be a division by 1000. This is fixed in #4, and only one exceptional case remains stored in milliseconds, the update interval.